src/Pages/ContentListTable/components/SnapshotDetailsModal/SnapshotDetailsModal.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  Stack,
+  StackItem,
+  Tab,
+  TabTitleText,
+  Tabs,
+} from '@patternfly/react-core';
+import { InnerScrollContainer } from '@patternfly/react-table';
+import { ContentOrigin } from '../../../../services/Content/ContentApi';
+import { useNavigate, useParams } from 'react-router-dom';
+import useRootPath from '../../../../Hooks/useRootPath';
+import { useAppContext } from '../../../../middleware/AppContext';
+import { useState } from 'react';
+import { SnapshotPackagesTab } from './Tabs/SnapshotPackagesTab';
+import { createUseStyles } from 'react-jss';
+import { SnapshotSelector } from './SnapshotSelector';
+
+const useStyles = createUseStyles({
+  modalBody: {
+    padding: '24px 24px 0 24px',
+  },
+  topContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: '24px 0',
+  },
+});
+
+export default function SnapshotDetailsModal() {
+  const { contentOrigin } = useAppContext();
+  const classes = useStyles();
+  const { repoUUID, snapshotUUID } = useParams();
+  const rootPath = useRootPath();
+  const navigate = useNavigate();
+  const [activeTabKey, setActiveTabKey] = useState<string | number>(0);
+
+  const handleTabClick = (
+    _: React.MouseEvent<HTMLElement, MouseEvent>,
+    tabIndex: string | number,
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  const onClose = () =>
+    navigate(rootPath + (contentOrigin === ContentOrigin.REDHAT ? `?origin=${contentOrigin}` : ''));
+
+  const onBackClick = () => navigate(rootPath + `/${repoUUID}/snapshots`);
```

Comment:
this should probably be `/${REPOSITORIES_ROUTE}/${repoUUID}/snapshots` too, i think that's the last one

Reply:
Thanks for catching all of these!